lib/httpoison_mock.ex

```diff
@@ -135,11 +143,22 @@ defmodule Gitea.HTTPoisonMock do
   """
   def delete(url) do
     Logger.debug("Gitea.HTTPoisonMock.delete/1 #{url}")
+    # check delete request endpooints
+    cond do
+      # match delete org
+      url =~ "/orgs" ->
```

Comment:
Check what is the delete request (delete org or delete repo at the moment) and returns  response accordingly